libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +60,9 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+  ngOnDestroy() {
+    if (this.booksSubscription) {
+      this.booksSubscription.unsubscribe();
+    }
+  }
```

Comment:
Unsubscribed on ngDestroy() which resolved memory leak issues in the application.